src/services/statistics.service.ts

```diff
@@ -158,4 +161,90 @@ export default class StatisticsService {
       );
     }
   }
+
+  async exportStatisticsToExcel(
+    startDate: string,
+    endDate: string,
+  ): Promise<Buffer> {
+    const treatmentNames = await this.getAllTreatments();
+    const workbook = new ExcelJS.Workbook();
+    const worksheet = workbook.addWorksheet('Estadísticas');
+
+    worksheet.columns = [
+      { header: 'Tratamiento', key: 'treatment', width: 30 },
+      { header: 'Total Ingresos', key: 'totalEarnings', width: 20 },
+      { header: 'Total Servicios', key: 'totalServices', width: 20 },
+      {
+        header: 'Citas Completadas',
+        key: 'totalCompletedAppointments',
+        width: 20,
+      },
+      {
+        header: 'Citas Canceladas',
+        key: 'totalCanceledAppointments',
+        width: 20,
+      },
+    ];
+    const headerRow = worksheet.getRow(1);
+    headerRow.getCell('treatment').fill = {
+      type: 'pattern',
+      pattern: 'solid',
+      fgColor: { argb: 'FFD9D9D9' }, // Gris claro
```

Comment:
💭 dev comments are in inglish

```suggestion
      fgColor: { argb: 'FFD9D9D9' }, // soft gray
```

Reply:
Suggestion applied, thanks!